### PR TITLE
Split account ID hashing / JWT secrets

### PR DESCRIPTION
**Pull request by FedericoCeratto**
Also fix tests and update the ClickHouse version

### CODE REVIEW

File: newapi/tests/integ/clickhouse_1_schema.sql [hunk 1]
```diff
@@ -28,7 +28,12 @@ CREATE TABLE default.fastpath
     `page_len_ratio` Float32,
     `server_cc` String,
     `server_asn` Int8,
-    `server_as_name` String
+    `server_as_name` String,
+    `test_version` String,
+    `test_runtime` Float32,
+    `architecture` String,
+    `engine_name` String,
+    `engine_version` String
```
Comment (bassosimone) [inline newapi/tests/integ/clickhouse_1_schema.sql:36]: I thought this diff had already been committed. If not, it strikes me as leaving the camp cleaner, so 👍

**Review by bassosimone — APPROVED** [review]
🐳

File: newapi/tests/integ/clickhouse_1_schema.sql (hunk 1 above)
Comment (FedericoCeratto) [reply newapi/tests/integ/clickhouse_1_schema.sql:36]: That was in the fastpath repo.

**Review by hellais — APPROVED** [review]
LGTM!

**Comment by FedericoCeratto** [comment]
1.0.41~pr321-551 deployed on FSN

**Pull request merged**